extensions/advanced_ballistics/AdvancedBallistics.cpp

```diff
@@ -94,7 +94,7 @@ double calculateAirDensity(double temperature, double pressure, double relativeH
 
     if (relativeHumidity > 0) {
         double _pSat = 6.1078 * pow(10, ((7.5 * temperature) / (temperature + 237.3)));
```

Comment:
Shouldn't 6.1078 (hPa) be converted to Pa directly here instead of converting it in `vaporPressure`? I find it more clear than later multiplying by 100.0 and having _`pSat` in hPa while `vaporPressure` in Pa